packages/picker/README.md

```diff
@@ -223,67 +137,78 @@ When you choose to leverage `<sp-menu-item>` elements without text content, you
 </sp-picker>
 ```
 
-### Advanced icon management
+##### Advanced icon management
 
-The `icons` attribute allows you to manage whether to `only` display the icon in the `<sp-picker>` element or to display `none` of the icons in the `<sp-picker>`.
+The `icons` attribute allows you to manage how the selected item will appear. Set `icons` to `only` display only the selected item's icon in the `<sp-picker>` element or `none` to display the selected item text without the icon `<sp-picker>`.
 
 When using `icons="only"` and your `<sp-menu-item>` elements still have text content, that content will be applied to `<sp-picker>` element in a non-visible way.
 
-```html
+<sp-tabs selected="only" auto label="Icons Attribute Options">
+<sp-tab value="only">Only</sp-tab>
+<sp-tab-panel value="only">
+
+```html demo
 <sp-field-label for="picker-icons-value">Choose an action...</sp-field-label>
 <sp-picker
     label="What would you like to do?"
-    value="item-2"
+    value="save"
     id="picker-icons-value"
     icons="only"
 >
-    <sp-menu-item>
+    <sp-menu-item value="save">
         <sp-icon-save-floppy slot="icon"></sp-icon-save-floppy>
         Save
     </sp-menu-item>
-    <sp-menu-item>
+    <sp-menu-item value="finish">
         <sp-icon-stopwatch slot="icon"></sp-icon-stopwatch>
         Finish
     </sp-menu-item>
-    <sp-menu-item>
+    <sp-menu-item value="review">
         <sp-icon-user-activity slot="icon"></sp-icon-user-activity>
         Review
     </sp-menu-item>
 </sp-picker>
 ```
 
-When using `icons="none"`, the icons will only be available in the overlaid menu.
+</sp-tab-panel>
+<sp-tab value="none">None</sp-tab>
+<sp-tab-panel value="none">
 
-```html
+```html demo
 <sp-field-label for="picker-icons-none">Choose an action...</sp-field-label>
 <sp-picker
     label="What would you like to do?"
-    value="item-2"
+    value="save"
     id="picker-icons-none"
     icons="none"
 >
-    <sp-menu-item>
+    <sp-menu-item value="save">
         <sp-icon-save-floppy slot="icon"></sp-icon-save-floppy>
         Save
     </sp-menu-item>
-    <sp-menu-item>
+    <sp-menu-item value="finish">
         <sp-icon-stopwatch slot="icon"></sp-icon-stopwatch>
         Finish
     </sp-menu-item>
-    <sp-menu-item>
+    <sp-menu-item value="review">
         <sp-icon-user-activity slot="icon"></sp-icon-user-activity>
         Review
     </sp-menu-item>
 </sp-picker>
 ```
 
-## Value
+</sp-tab-panel>
+</sp-tabs>
+
+#### Value
 
 When the `value` of an `<sp-picker>` matches the `value` attribute or the trimmed `textContent` (or `itemText`) of a descendent `<sp-menu-item>` element, it will make that element as `selected`.
```

Comment:
```suggestion
When the `value` of an `<sp-picker>` matches either the `value` attribute or the trimmed `textContent` (or `itemText`) of a descendent `<sp-menu-item>`, it will mark that element as `selected`.
```